<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResizeRecipeDialog</class>
 <widget class="QWidget" name="ResizeRecipeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>174</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KButtonGroup" name="m_buttonGroup">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_yieldButton">
        <property name="text">
         <string>&amp;Scale by yield</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_yieldFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="1">
          <widget class="KLineEdit" name="m_currentYieldInput">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;New yield</string>
           </property>
           <property name="buddy">
            <cstring>m_newYieldInput</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Current &amp;yield:</string>
           </property>
           <property name="buddy">
            <cstring>m_currentYieldInput</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="FractionInput" name="m_newYieldInput"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_factorButton">
        <property name="text">
         <string>Scale b&amp;y factor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="m_factorFrame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Factor (e.g. 1/2 to half, 3 to triple):</string>
           </property>
           <property name="buddy">
            <cstring>m_factorInput</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="FractionInput" name="m_factorInput"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>FractionInput</class>
   <extends>QLineEdit</extends>
   <header>widgets/fractioninput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
